src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -139,6 +160,21 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
       addListener("hint", (newHint: any) => {
         setNewHint(newHint.text || "");
       });
+      addListener("decoratedContentEvent", (msg: IDecoratedContentEvent) => {
+        const { textDecorationHandlerInfo } = pluginInfo;
+        if (textDecorationHandlerInfo) {
+          if ("type" in textDecorationHandlerInfo.eventListeners && textDecorationHandlerInfo.eventListeners.type === msg.type) {
```

Comment:
You could combine the array and non-array paths with something like:
```typescript
          const listeners = Array.isArray(textDecorationHandlerInfo.eventListeners)
							  ? textDecorationHandlerInfo.eventListeners
							  : [textDecorationHandlerInfo.eventListeners];
		  listeners.forEach(listener => ...
```
there's a lodash utility called `castArray` that will do the same thing.

Reply:
Agreed.  This is cleaner.  Fixed.